kotlin/workflow-runtime/src/test/java/com/squareup/workflow/internal/RealRenderContextTest.kt

```diff
@@ -209,6 +214,7 @@ class RealRenderContextTest {
     val childCases = context.buildBehavior()
         .childCases
     assertEquals(1, childCases.size)
+    // IDE is lying, this is fine.
```

Comment:
Was there a warning of some sort? I'm not seeing it :)

Reply:
Type inference failed error. 🤷‍♂

Reply:
Did you have new inference enabled?